TheaterDatabase/dbo/Tables/Performance.sql

```diff
@@ -0,0 +1,21 @@
+﻿CREATE TABLE [dbo].[Performance] (
+    [PerformanceId] INT identity(1,1) NOT NULL,
+	[Image]    VARBINARY (MAX) NOT NULL,
+    [Duration]      INT  NOT NULL,
+    [MinPrice]      INT  NOT NULL,
+    [MaxPrice]      INT  NOT NULL,
+    [MinimumAge]    INT  NOT NULL,
+    [Description]   TEXT NULL,
+    [MainImageId]   INT  NOT NULL,
+    [ProducerId]    INT  NULL,
+    [AuthorId]      INT  NULL,
```

Comment:
What do you think about creating table PerformanceCreativeTeamMember with column Role
This will allow many to many relationship with setting of role for each person